dkg-gadget/src/utils.rs

```diff
@@ -151,34 +140,6 @@ where
 	false
 }
 
-pub(crate) fn fetch_public_key<B, C, BE>(dkg_worker: &mut DKGWorker<B, C, BE>) -> Public
-where
-	B: Block,
-	BE: Backend<B>,
-	C: Client<B, BE>,
-	C::Api: DKGApi<B, AuthorityId, <<B as Block>::Header as Header>::Number>,
-{
-	dkg_worker
-		.key_store
-		.authority_id(&dkg_worker.key_store.public_keys().unwrap())
-		.unwrap_or_else(|| panic!("Halp"))
-}
-
-pub(crate) fn fetch_sr25519_public_key<B, C, BE>(
-	dkg_worker: &mut DKGWorker<B, C, BE>,
-) -> sp_core::sr25519::Public
-where
-	B: Block,
-	BE: Backend<B>,
-	C: Client<B, BE>,
-	C::Api: DKGApi<B, AuthorityId, <<B as Block>::Header as Header>::Number>,
-{
-	dkg_worker
-		.key_store
-		.sr25519_authority_id(&dkg_worker.key_store.sr25519_public_keys().unwrap_or_default())
-		.unwrap_or_else(|| panic!("Could not find sr25519 key in keystore"))
-}
-
 pub fn get_key_path(base_path: &Option<PathBuf>, path_str: &str) -> Option<PathBuf> {
```

Comment:
It would be good to document the public functions within this file. 

- [ ] `get_key_path`
- [ ] `set_up_rounds`
- [ ] `validate_threshold`
- [ ] `find_index`